### PR TITLE
Hockey App Integration

### PR DESCRIPTION
**Pull request by brian-norman**
**Instructions**:

You need to add the `AppCenterAPIKey` to your `gradle.properties` file now, which is just adding a line:
`AppCenterAPIKey = "LONG-KEY-STRING"`

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -60,4 +60,10 @@ dependencies {
     androidTestImplementation 'com.android.support.test:rules:1.0.2'
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'androidx.test.espresso:espresso-contrib:3.1.1'
+
+    // Visual Studio App Center
+    def appCenterSdkVersion = '2.0.0'
```
Comment (wesley-tang) [inline app/build.gradle:68]: Does Java constant convention not apply here?

Also is there a way to make this a constant?

Comment (wesley-tang) [reply app/build.gradle:68]: Oh wait is def the version of constant?

Comment (wesley-tang) [reply app/build.gradle:68]: So like APP_CENTER_SDK_VERSION

File: app/src/main/java/com/cobresun/factfulnewsandroid/ui/activities/MainActivity.kt [hunk 2]
```diff
@@ -17,6 +22,11 @@ class MainActivity : AppCompatActivity() {
         val toolbar = findViewById<Toolbar>(R.id.toolbar)
         setSupportActionBar(toolbar)
 
+        AppCenter.start(
+            application, "40e7957a-3008-49b7-af7b-5bf1be103ad5",
```
Comment (wesley-tang) [inline app/src/main/java/com/cobresun/factfulnewsandroid/ui/activities/MainActivity.kt]: This magic number would do well being defined explicitly or even hidden possibly?

**Review by wesley-tang — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (brian-norman) [reply app/build.gradle:68]: Gradle convention is to write it in camelCase, the code you're looking at is from Microsoft documentation

File: app/src/main/java/com/cobresun/factfulnewsandroid/ui/activities/MainActivity.kt (hunk 2 above)
Comment (brian-norman) [reply app/src/main/java/com/cobresun/factfulnewsandroid/ui/activities/MainActivity.kt]: Good idea

**Review by SunnyNagam — APPROVED** [review]

**Review by wesley-tang — APPROVED** [review]
thanks!

**Pull request merged**